Modules/Microsoft365DSC/Examples/Resources/IntuneSecurityBaselineDefenderForEndpoint/1-Create.ps1

```diff
@@ -37,9 +37,9 @@ Configuration Example
                 DisableSafetyFilterOverrideForAppRepUnknown = '1'
             }
             Ensure                = 'Present'
-            ApplicationId         = $ApplicationId;
-            TenantId              = $TenantId;
-            CertificateThumbprint = $CertificateThumbprint;
+            ApplicationId         = $ApplicationId
+            TenantId              = $TenantId
+            CertificateThumbprint = $CertificateThumbprint
```

Comment:
Should not be part of this PR, but since it's here... All examples should have the values ending with a semi-colon since that's how M365DSC does exports.

Reply:
True, this was a quick update that I spotted while copying these lines. Since PowerShell does not need semi-colons, my preference is to remove them....it is just clutter. 

Because of the C# background of Nik (where the semi-colons are required) he has added the semi-colons to the export 😉, but I would suggest to remove it there as well

Reply:
I understand but one of the tests I have created is that the resulting blueprint of my converter ends up as-is of what M365DSC would export, in this case *Get-M365DSCExportContentForResource*.

My unit tests also make sure that *ConvertTo-DSCObject* behaves as it should.

Reply:
What is your suggestion here? Should we add a semi-colon to all examples or remove them everywhere and make sure the export also doesn't use semi-colons?

Reply:
And maybe it is an idea to integrate some of the tests you created into the module....that would probably make the quality and predictability of the module much better.